app/src/main/java/zegreanu/cristi/geoquiz/CheatActivity.kt

```diff
@@ -0,0 +1,40 @@
+package zegreanu.cristi.geoquiz
+
+import android.content.Intent
+import android.os.Bundle
+import android.widget.Button
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import kotlin.properties.Delegates
+
+class CheatActivity : AppCompatActivity() {
+    private val EXTRA_ANSWER_IS_TRUE = "zegreanu.cristi.geoquiz.answer_is_true"
+    private val EXTRA_ANSWER_SHOWN = "zegreanu.cristi.geoquiz.answer_shown"
+
+    private lateinit var showAnswerButton: Button
+    private lateinit var answerTextView: TextView
+
+    private var answerIsTrue by Delegates.notNull<Boolean>()
```

Comment:
Since we're going to move the `newIntent` method here, it will be somehow obvious that the way you open this activity is by passing a boolean argument, so it's fair to default this to false and read it from the bundle. It already defaults to `false` when we read it by passing `false` to `intent.getBooleanExtra(EXTRA_ANSWER_IS_TRUE, false)`.